smoke-test/src/test/java/org/opennms/smoketest/IntegrationAPIIT.java

```diff
@@ -48,21 +48,36 @@
     private static final Logger LOG = LoggerFactory.getLogger(IntegrationAPIIT.class);
 
     @ClassRule
-    public static OpenNMSStack stack = OpenNMSStack.MINIMAL;
+    public static OpenNMSStack stack = OpenNMSStack.MINION;
 
     @Test
     public void canLoadSampleProject() throws Exception {
-        final InetSocketAddress karafSsh = stack.opennms().getSshAddress();
-        // Install the sample project
-        try (final SshClient sshClient = new SshClient(karafSsh, "admin", "admin")) {
+        // Install the sample plugin on OpenNMS
+        final InetSocketAddress opennmsSsh = stack.opennms().getSshAddress();
+        try (final SshClient sshClient = new SshClient(opennmsSsh, "admin", "admin")) {
             PrintStream pipe = sshClient.openShell();
-            pipe.println("feature:install opennms-integration-api-sample-project");
+            pipe.println("feature:install opennms-plugin-sample");
             pipe.println("logout");
             await().atMost(1, MINUTES).until(sshClient.isShellClosedCallable());
         }
 
-        // Now wait until the health check passes
-        verifyHealthCheckWithDescription(karafSsh, "Sample Project :: Health Check");
+        // Install the sample plugin on Minion
+        final InetSocketAddress minionSsh = stack.minion().getSshAddress();
+        try (final SshClient sshClient = new SshClient(minionSsh, "admin", "admin")) {
+            PrintStream pipe = sshClient.openShell();
+            pipe.println("feature:install minion-plugin-sample");
+            pipe.println("logout");
+            await().atMost(1, MINUTES).until(sshClient.isShellClosedCallable());
+        }
+
+        // Now wait until the health check passes on OpenNMS
+        verifyHealthCheckWithDescription(opennmsSsh, "Sample Project :: Health Check");
+
+        // Now do the same on Minion
+        verifyHealthCheckWithDescription(minionSsh, "Sample Project :: Minion");
+
+        // Now ensure that we're able to invoke the extensions on Minion from OpenNMS
+        verifyHealthCheckWithDescription(opennmsSsh, "Sample Project :: Service Extensions on Minion");
```

Comment:
This verifies that the extensions can actually be invoked on Minion and tests the round-trip.